boilerplate/src/app/_layout.tsx

```diff
@@ -27,7 +27,8 @@ export default function Root() {
 
   const [fontsLoaded, fontError] = useFonts(customFontsToLoad)
 
-  const loaded = fontsLoaded && rehydrated
+  const loaded = fontsLoaded 
+                         && rehydrated // @mst remove-current-line
```

Comment:
I did a double-take at this and wondered if it would pass lint check if we don't remove MST, and then I remembered we run a lint fix after that processing. All good 👍

Reply:
Yeah! Looks awkward but I didn't want to have to make some eslint type directive of
 `const loaded = fontsLoaded /* @mst remove-start */ &&  rehydrated /* @mst remove-end */`
 
 Although we do have that already right? can you swap `//` for `/*`? I'll have to check that now

Reply:
I tried using `  const loaded = fontsLoaded /* @mst remove-block-start */ && rehydrated /* @mst remove-block-end */`

but the entire line gets removed